src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -3066,37 +3056,37 @@ TEST_P(conv_int8_eltwise_onednn, u8_eltwise_prod_out) {
         data("scale_data", get_mem(get_per_channel_layout(p), 1.0f/p.kernel.count()) ),
         convolution("conv_prim", input_info("input"), { "weights" }, { "bias" }, p.groups, p.stride, p.pad, p.dilation),
         eltwise("scale", { input_info("conv_prim"), input_info("scale_data") }, eltwise_mode::prod, data_types::u8),
-        crop("crop", input_info("scale"), get_output_layout(p).get_tensor(), { 0, 0, 0, 0 }),
-        reorder("reorder_bfyx", input_info("crop"), p.default_format, data_types::f32)
+        reorder("reorder_bfyx", input_info("scale"), p.default_format, data_types::f32)
     );
 
     tolerance = 1.f;
     execute(p);
 }
 
+static const int NOT_FOR_CLDNN=0;
```

Comment:
nit: `NOT_FOR_CLDNN = 0` (space before and after parenthesis)

Reply:
Done.